pkg/gooq/expression.go

```diff
@@ -300,6 +305,14 @@ func (expr *expressionImpl) Render(
 		builder.RenderExpression(lhs).
 			Print(" ").Print(expr.operator.String()).Print(" ").
 			RenderExpression(rhs)
+	case ExpressionTypeMultigrade:
```

Comment:
not sure what to call this but apparently it is called multigrade - https://en.wikipedia.org/wiki/Plural_quantification
```
e.g. Alice and Bob cooperate.
Alice, Bob and Carol cooperate.
```
In our case what I want is 
```
boolExpr1 AND boolExpr2 AND boolExpr3
boolExpr1 OR boolExpr2 OR boolExpr3
```
Could also call this `ExpressionTypeJoin` but that is also weird